packages/odyssey-storybook/src/components/odyssey-mui/Toast/Toast.stories.tsx

```diff
@@ -68,13 +74,34 @@ const DefaultTemplate: Story = (args) => {
   return (
     <>
       <Button variant="primary" onClick={handleClick}>
-        Open {args.severity} snackbar
+        Open {args.severity} toast
       </Button>
       <Stack spacing={2} sx={{ width: "100%" }}>
-        <Snackbar open={open} autoHideDuration={6000} onClose={handleClose}>
-          <Alert severity={args.severity} variant="toast">
-            {args.title && <AlertTitle>{args.title}</AlertTitle>}
-            {args.content}
+        <Snackbar
+          open={open}
+          autoHideDuration={args.isDismissible === true ? undefined : 6000}
+          onClose={handleClose}
+        >
+          <Alert
+            severity={args.severity}
+            variant="toast"
+            action={
+              args.isDismissible && (
+                <Button
+                  aria-label="close"
+                  onClick={() => {
+                    setOpen(false);
+                  }}
```

Comment:
I dunno how we want to handle functions like this in stories.

This is completely fine in a story, but a dev who copies it will then have bad code. Do we wanna use `useCallback` to show best practices? I'm not sure what shows up when clicking "Show Code".

Reply:
No strong opinion here. 

My current thinking is that post-1.0, anything that isn't a wrapped+exported component ready for us would be in `odyssey-labs`, so a lot of my work that looks like this would live there. I'm not _really_ a JS eng, so I like being able to "make it work" when we're still just using stories for scaffolding. TBH devs probably shouldn't copy any of my JS...

Reply:
```jsx
  const openToast = (() => {
    setOpen(true);
  }, []);
  const closeToast = useCallback(() => {
    setOpen(false);
  }, []);
  return (
    <>
      <Button variant="primary" onClick={openToast}>
        Open {args.severity} snackbar
        Open {args.severity} toast
      </Button>
      <Stack spacing={2} sx={{ width: "100%" }}>
        <Snackbar open={open} autoHideDuration={6000} onClose={handleClose}>
          <Alert severity={args.severity} variant="toast">
            {args.title && <AlertTitle>{args.title}</AlertTitle>}
            {args.content}
        <Snackbar
          open={open}
          autoHideDuration={args.isDismissible === true ? undefined : 6000}
          onClose={closeToast}
        >
          <Alert
            severity={args.severity}
            variant="toast"
            action={
              args.isDismissible && (
                <Button
                  aria-label="close"
                  onClick={closeToast}
```

Reply:
Getting a `Cannot find name 'useCallback'.`

Reply:
Import `useCallback` from React.